Fri Dec 22 23:28:44 2023       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 545.23.08              Driver Version: 545.23.08    CUDA Version: 12.3     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  Tesla V100-SXM2-32GB           On  | 00000000:8A:00.0 Off |                    0 |
| N/A   32C    P0              40W / 300W |      0MiB / 32768MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
0 model/Geoformer_beforeTrans.pkl
1 model/5x5p_1.pkl
Traceback (most recent call last):
  File "/glade/work/zilumeng/3D_trans/./Code/free_run_1.py", line 56, in <module>
    mymodel.load_state_dict(torch.load(adr))
  File "/glade/work/zilumeng/conda-envs/earth2mip/lib/python3.10/site-packages/torch/nn/modules/module.py", line 2041, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for Geoformer:
	size mismatch for predictor_emb.emb_space.weight: copying a param with shape torch.Size([240, 256]) from checkpoint, the shape in current model is torch.Size([510, 256]).
	size mismatch for predictand_emb.emb_space.weight: copying a param with shape torch.Size([240, 256]) from checkpoint, the shape in current model is torch.Size([510, 256]).
